docs/pages/deployment/security-considerations.rst

```diff
@@ -0,0 +1,37 @@
+.. _security-considerations::
+
+Security Considerations
+#######################
```

Comment:
[did:web](https://w3c-ccg.github.io/did-method-web/#dns-considerations) recommends? [DNS over HTTPS](https://datatracker.ietf.org/doc/html/rfc8484) and DNSSEC [RFC4033](https://tools.ietf.org/html/rfc4033), [RFC4034](https://tools.ietf.org/html/rfc4034), and [RFC4035](https://tools.ietf.org/html/rfc4035).

[openid4vp](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-normative-references) requires [BCP195](https://www.rfc-editor.org/info/bcp195)

I think we should therefore include these in our recommendations/requirements as well